src/components/ArticleCard.jsx

```diff
@@ -1,7 +1,9 @@
+import { Link} from "react-router-dom"
+
 const ArticleCard = ({article}) => {
 return (
     <div className="articleCard">
-        <h3>{article.title}</h3>
+        <Link to={`/articles/${article.article_id}`}><h3>{article.title}</h3></Link>
         <p className="articleCardInfo"><b>Author: </b>{article.author}
         <br></br>
         <b>Posted:</b> {article.created_at}
```

Comment:
Good work so far - see this article for more interesting ways of displaying information: https://www.refactoringui.com/previews/labels-are-a-last-resort